docs/common/env_util.rst

```diff
@@ -0,0 +1,7 @@
+.. _env_util:
+
+Command Utils
+=========================
+
+.. automodule:: stable_baselines3.common.env_util
```

Comment:
btw, why `env_util` and not `env_utils` ?

Reply:
I looked at `save_util.py` naming for this one (but granted, we have `utils.py`). I say we use `env_util` for now to stay consistent with `save_util`.